[error] insecure HTTP request is unsupported 'http://repo.artima.com/releases'; switch to HTTPS or opt-in as ("Artima Maven Repository" at "http://repo.artima.com/releases").withAllowInsecureProtocol(true), or by using allowInsecureProtocol in repositories file
[error] java.lang.RuntimeException: insecure protocol is unsupported
[error] 	at scala.sys.package$.error(package.scala:30)
[error] 	at sbt.Classpaths$.errorInsecureProtocol(Defaults.scala:3301)
[error] 	at sbt.coursierint.LMCoursier$.coursierConfiguration(LMCoursier.scala:169)
[error] 	at sbt.coursierint.LMCoursier$.$anonfun$scalaCompilerBridgeConfigurationTask$1(LMCoursier.scala:308)
[error] 	at sbt.coursierint.LMCoursier$$$Lambda$426/0x00000000f061bff8.apply(Unknown Source)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at scala.Function1$$Lambda$329/0x0000000094c0e8a8.apply(Unknown Source)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error] 	at sbt.internal.util.$tilde$greater$$Lambda$2640/0x000000009586d650.apply(Unknown Source)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error] 	at sbt.Execute$$Lambda$2665/0x0000000095875698.apply(Unknown Source)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error] 	at sbt.Execute.work(Execute.scala:291)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error] 	at sbt.Execute$$Lambda$2650/0x00000000958716d0.apply(Unknown Source)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.ConcurrentRestrictions$$anon$4$$Lambda$2663/0x00000000958736a0.apply(Unknown Source)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
[error] 	at java.base/java.lang.Thread.run(Thread.java:853)
[error] (scalaCompilerBridgeScope / csrConfiguration) insecure protocol is unsupported
